bench/wtperf/wtperf.h

```diff
@@ -223,6 +256,10 @@ struct __config_thread {		/* Per-thread structure */
 	TRACK insert;			/* Insert operations */
 	TRACK read;			/* Read operations */
 	TRACK update;			/* Update operations */
+	TRACK truncate;			/* Truncate operations */
+	TRACK truncate_sleep;		/* Truncate sleep operations */
+
+	uint64_t total_inserts;		/* Inserts done by this thread */
```

Comment:
p.s: The ```TRACK``` code in ```wtperf``` is a bit esoteric, but it's worth taking the time to figure out what it's doing.